### DISCUSSION TITLE
The exact time point of the aphelion

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2968] [Q&A]
I am looking for the exact time point of the ephelion.
and the position in celestial coordinates of the sun at that point in time.

**Comment by gzotti** [comment]
Stellarium does not provide this (first) kind of information.

**Comment by tzmosh** [comment]
In the Astronomical calculations window, Under the Phenomena tab, the perihelion and aphelion option appears. I don't see that it appears in the results of the calculations.
